### PR TITLE
Platypus

### PR DESCRIPTION
**Pull request by afraz-ahmed**
For INC1046171

Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonPackage'
```
Comment (edmondac) [inline easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb:1]: Shouldn't this be a `MakeCp`, looking at https://github.com/andyrimmer/Platypus/blob/master/Makefile

In which case various lines in the rest of the file would change.

Comment (afraz-ahmed) [reply easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb:1]: In https://github.com/andyrimmer/Platypus/blob/master/release/AllReleases/Platypus_0.8.1.tgz, there is no Makefile.

The easyconfig is from upstream, with a few small additions/changes.

Comment (afraz-ahmed) [reply easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb:1]: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-intel-2016a-Python-2.7.11.eb

Comment (edmondac) [reply easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-GCCcore-8.3.0-Python-2.7.16.eb:1]: Oh, the released tgz contains a setup.py that seems to do the compiling. Odd that it's so different from the repo.

**Pull request merged**